README.md

```diff
@@ -1,4 +1,4 @@
-# PDOK storage usage exporter
+# Azure storage usage exporter
```

Comment:
Het leest nog erg alsof het heel PDOK specifiek is. Maar het lijkt mij nu volledig generiek (?). In principe zou iedereen met Azure Blob hiermee z'n storage usage kunnen monitoren. Voorstel voor aangepaste tekst (ter vervanging van huidige inleiding):

> This Prometheus exporter exposes statistics about [Azure Blob storage](https://azure.microsoft.com/en-us/products/storage/blobs) usage.
It relies upon data from [Azure Storage blob inventory reports](https://learn.microsoft.com/en-us/azure/storage/blobs/blob-inventory). This data is aggregated, matched against configured labels and  exposed as a Prometheus metrics endpoint. The goal is to expose stats about storage/disk usage (not transactions) per Azure Blob container/directory/prefix.